app/views/support_interface/application_forms/application_choices/confirm_reinstate_offer.html.erb

```diff
@@ -0,0 +1,47 @@
+<% content_for :browser_title, title_with_error_prefix('Reinstate offer', @declined_course_choice.errors.any?) %>
+<% content_for :before_content, govuk_back_link_to(support_interface_application_form_path(application_form_id: params[:application_form_id]), 'Back to application') %>
+
+<%= form_with(
+  model: @declined_course_choice,
+  url: support_interface_application_form_reinstate_offer_path(application_form_id: params[:application_form_id], application_choice_id: params[:application_choice_id]),
+  method: :patch
+  )do |f| %>
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <%= f.govuk_error_summary %>
+      <h1 class="govuk-heading-l">
+        Are you sure you want to reinstate this offer?
+      </h1>
+
+      <p class="govuk-body">An offer can only be reinstated if:</p>
+      <ul class="govuk-list govuk-list--bullet">
+        <li>it was declined within 5 working days of the request to reinstate and,</li>
+        <li>the candidate has not accepted any other offers</li>
+      </ul>
+      <p class="govuk-body">In order to reinstate the offer you must first contact the provider to confirm that they agree to this.</p>
+      <p class="govuk-body">By reinstating the offer, the DBD deadline for this offer (and any other offers) will be reset to 10 working days.</p>
+      <p class="govuk-body">Once the offer has been reinstated, please email the candidate using the macro.</p>
+      <p class="govuk-body">There are separate macros if the request was made after 5 working days or if the candidate has already accepted another offer.</p>
+    </div>
+  </div>
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <%= f.govuk_text_field(
+              :audit_comment_ticket,
+              label: {
+                text: 'Zendesk ticket URL',
+                size: 'm',
+              },
+              rows: 1,
+              hint: { text: 'For example https://becomingateacher.zendesk.com/agent/tickets/12345' },
+            ) %>
+    </div>
+  </div>
+
+  <%= f.govuk_check_boxes_fieldset :accept_guidance, legend: { text: 'Terms of use', size: 'm' } do %>
+    <%= f.govuk_check_box :accept_guidance, true, multiple: false, label: { text: 'I have read the guidance' }, link_errors: true %>
+  <% end %>
```

Comment:
Maybe it’s just a case of removing the fieldset?

```suggestion
  <%= f.govuk_check_box :accept_guidance, true, multiple: false, label: { text: 'I have read the guidance' }, link_errors: true %>
```

Reply:
Hmmm 🤔  So I just tried without the fieldset - it no longer displays the validation error within that section (as it does with the URL):
![image](https://user-images.githubusercontent.com/47917431/116688733-039e2980-a9af-11eb-924a-6a7f1c842a19.png)

Reply:
To hide the text I can set  `legend: { text: ' ', size: 'm' } do` 🤷‍♂️ 
![image](https://user-images.githubusercontent.com/47917431/116691118-48778f80-a9b2-11eb-98dc-5d612be6f0a7.png)

Reply:
What happens if you set `legend: nil`?